crates/e2e/tests/e2e/refunder.rs

```diff
@@ -90,22 +93,19 @@ async fn refunder_tx(web3: Web3) {
     // Run autopilot indexing loop
     services.maintenance.run_maintenance().await.unwrap();
 
-    // The blockchain is in the past for this test. We validate this assumption here, if this assertion fails then the
-    // original test conditions should be restored.
-    let now = chrono::offset::Utc::now().timestamp();
-    assert!(
-        (valid_to as i64) < now,
-        "Order should be expired from the point of view of the refunder."
-    );
```

Comment:
I'm happy to see this go! 
Consider also changing `test_node_in_the_past` to `test` at the start of the test. (We should probably revert the new test infrastructure from #851.)